docs/reference/commands.rst

```diff
@@ -18,5 +18,6 @@ inmanta
 
 inmanta-cli
 ###########
-
-Use ``inmanta-cli --help``
+.. click:: inmanta.main:cmd
```

Comment:
We should also add a small description of the `inmanta-cli` command as is done for the `inmanta` command. This would also solve the ugliness of the two inmanta-cli headers rights after each other.